internal/commands/ostest/depgraph_flow.go

```diff
@@ -47,9 +47,11 @@ func RunUnifiedTestFlow(
 ) ([]workflow.Data, error) {
 	ictx := cmdctx.Ictx(ctx)
 	logger := cmdctx.Logger(ctx)
+	progressBar := cmdctx.ProgressBar(ctx)
 
 	logger.Info().Msg("Starting open source test")
 
+	progressBar.SetTitle("Computing dependency graph")
```

Comment:
Sounds quite technical... should we make it `"Analyzing dependencies..."/"Listing dependencies..."`? Btw. I wonder if we should add a `"..."` to the end of the status text. Maybe we need to include content/design here? 🙃